spec/rubocop/cop/rails/active_record_override_spec.rb

```diff
@@ -84,4 +84,16 @@ def save
       RUBY
     end
   end
+
+  context 'when class has no parent specified' do
+    it 'registers no offense when overriding save' do
```

Comment:
I don’t understand the description – this class does *not* override `#save`…

Reply:
Nice catch!  I [updated](https://github.com/rubocop-hq/rubocop/compare/c9fda9992d25242d67e624cace89dd2428a6ba5e..ae18831e12c33bee58d560dff831838d70fe48e4) to actually be `save` (for consistency with the other tests in there).